README.md

```diff
@@ -491,6 +525,26 @@ Generates the following response payload:
 
 All 500 errors hide your message from the end user. Your message is recorded in the server log.
 
+### `Boom.badImplementation([message], [data])`
```

Comment:
I sorted the methods by http status code in ascending order, thus resulting in the move here.